CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 All notable changes to this project will be documented in this file based on the
 [Keep a Changelog](http://keepachangelog.com/) Standard.
 
-## [Unreleased](https://github.com/elastic/packetbeat/compare/1.0.0-beta3...HEAD)
```

Comment:
I suggest to always leave the "unrelease" header plus the subtitles in. So this line would be now:
    
    ## [Unreleased](https://github.com/elastic/packetbeat/compare/1.0.0-beta4...HEAD)